### PR TITLE
adds ETO menu to layout

### PR DESCRIPTION
**Pull request by Moejoe90**
image

### CODE REVIEW

File: app/components/layouts/LayoutAuthorizedMenu.tsx [hunk 1]
```diff
@@ -38,11 +45,43 @@ const MenuEntry: React.SFC<IMenuEntry & NavLinkProps> = ({
...
-export const LayoutAuthorizedMenu = () => (
-  <div className={styles.menu}>
-    <MenuEntry svgString={iconStart} to={appRoutes.dashboard} title="Start" />
-    <MenuEntry svgString={iconPortfolio} to="#" title="Portfolio" />
-    <MenuEntry svgString={iconWallet} to={appRoutes.wallet} title="Wallet" />
-    <MenuEntry svgString={iconSettings} to={appRoutes.settings} title="Settings" actionRequired />
-  </div>
-);
+export const LayoutAuthorizedMenuComponent = ({ userType }: IStateProps) => {
+  if (userType === "investor")
```
Comment (krzkaczor) [inline app/components/layouts/LayoutAuthorizedMenu.tsx]: could you use switch statement here instead (on default it should throw)

File: app/components/layouts/LayoutAuthorizedMenu.tsx [hunk 2]
```diff
@@ -38,11 +45,43 @@ const MenuEntry: React.SFC<IMenuEntry & NavLinkProps> = ({
...
+    return (
+      <div className={styles.menu} data-test-id="investor-menu">
```
Comment (krzkaczor) [inline app/components/layouts/LayoutAuthorizedMenu.tsx]: export please these menus to separate components. Can be in the same file. Like InvestorsMenu & IssuerMenu

File: app/components/layouts/LayoutAuthorizedMenu.tsx [hunk 3]
```diff
@@ -38,11 +45,38 @@ const MenuEntry: React.SFC<IMenuEntry & NavLinkProps> = ({
...
+    case "investor":
+      return <InvestorMenu data-test-id="investor-menu" />;
+    case "issuer":
+      return <IssuerMenu data-test-id="issuer-menu" />;
+    default:
+      throw new Error("Menu loaded in wrong state, user should be either issuer or investor");
+  }
+};
+
+export const LayoutAuthorizedMenu = appConnect<IStateProps, {}>({
```
Comment (krzkaczor) [inline app/components/layouts/LayoutAuthorizedMenu.tsx]: FYI: you dont need to provide a second type argument - `{}` its optional

Comment (Moejoe90) [reply app/components/layouts/LayoutAuthorizedMenu.tsx]: Hmmm for some reason the linter complains

**Pull request merged**